scoring_engine/engine/engine.py

```diff
@@ -217,7 +217,9 @@ def run(self):
                 logger.info("All jobs have finished for this round")
 
                 logger.info("Determining check results and saving to db")
-                round_obj = Round(number=self.current_round)
+                round_obj = Round(round_start=round_start_time, number=self.current_round)
+                round_end_time = datetime.now()
+                round_obj.round_end = round_end_time
```

Comment:
round start and end times were not being correctly populated in the database causing incorrect values shown on the /stats page. this fixes the issue